openmls/src/schedule/psk.rs

```diff
@@ -90,49 +109,33 @@ impl ExternalPsk {
     }
 }
 
-/// External PSK Bundle. This contains the secret part of the PSK as well as the
+/// Contains the secret part of the PSK as well as the
 /// public part that is used as a marker for injection into the key schedule.
-pub struct ExternalPskBundle {
+#[derive(Serialize, Deserialize)]
+pub struct PskBundle {
+    psk_id: PreSharedKeyId,
```

Comment:
Why do we need the psk id in here?
The only place I found where it is used it's not needed because we have the ids there.

Reply:
How else would you identify what secret to use? Note that this is just a more generalised version of what we had before (now other types of PSKs are supported, not just `ExternalPsk`).

Reply:
The only function that I see using this gets a list of `psk_id`s as input. Do we use it anywhere else?

Reply:
Well, the whole idea is to store this in the key store. Alternatively, we could only store the `Secret` in the key store and then the consumers have to bring their own out-of-band storage for `PreSharedKeyId`.

Reply:
Your description says "The KeyStore can now store and read PskBundle, referenced by PreSharedKeyId". So yes, you need to store the ID somewhere else. It's of no use having it in the stored value.

Reply:
This is aligned with the logic we use for `KeyPackageBundle` and `CredentialBundle`: We not only store the private key material in the key store, but also the public keys and other non-secret data. I don't see how this is any different.
Is your proposal to only store the `Secret`? I don't understand what exactly you are proposing.